awsconfigrules/Makefile

```diff
@@ -1,7 +1,15 @@
+ifdef ACCOUNT
+else
+        ACCOUNT := 442294194136
+        export
+endif
+
+TEST := false
 
 ifdef AWS_REGION
 else
-	AWS_REGION := us-east-1
+       AWS_REGION := us-east-1
```

Comment:
Shouldnt this no more be hardcoded? like use env variables

Reply:
the value is coming from env variable, this is default value, if env variable is missing.